services/frontend-v3/src/components/profileForms/employmentDataForm/EmploymentDataFormView.jsx

```diff
@@ -69,7 +69,7 @@ const EmploymentDataFormView = ({
   /* Component Styles */
   const styles = {
     skeleton: {
-      minHeight: "400px",
+      minHeight: "658px",
```

Comment:
please move these to separate PR. I like the change and i don't want to revert it in case the URL lag fix needs to be reverted